<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElementForm</class>
 <widget class="QWidget" name="ElementForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>632</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="tomo::widget::ToolchainComboBox" name="elementList"/>
   </item>
   <item>
    <widget class="tomo::widget::VerticalScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>239</width>
        <height>579</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="parameters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Parameters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="viewParameters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>View Parameters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="information">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Information</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="tbd::qt::ConfigTree" name="queryView" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tomo::widget::ToolchainComboBox</class>
   <extends>QComboBox</extends>
   <header>widget/ToolchainComboBox.h</header>
   <slots>
    <signal>elementSelected(tomo::base::Element*)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>tbd::qt::ConfigTree</class>
   <extends>QWidget</extends>
   <header location="global">tbd/qt/ConfigTree.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tomo::widget::VerticalScrollArea</class>
   <extends>QScrollArea</extends>
   <header>widget/VerticalScrollArea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>elementList</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ElementForm</receiver>
   <slot>setElement()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setElement()</slot>
 </slots>
</ui>
